include/core/utilities.h

```diff
@@ -241,9 +241,9 @@ announce_string(const ConditionalOStream &pcout,
                 const std::string         expression,
                 const char                delimiter = '-')
 {
-  pcout << std::string(expression.size() + 1, delimiter) << std::endl;
+  pcout << std::string(expression.size(), delimiter) << std::endl;
   pcout << expression << std::endl;
-  pcout << std::string(expression.size() + 1, delimiter) << std::endl;
+  pcout << std::string(expression.size(), delimiter) << std::endl;
```

Comment:
Personnaly I thought it was cleaner with the +1., but if you think it's better without i am ok with it.

Reply:
Another option would be `expression.size() + 1` + start the string with white space in front. Which one do you prefer?

Reply:
> Another option would be `expression.size() + 1` + start the string with white space in front. Which one do you prefer?

Nah I don't think starting with a white space is the right approach :)